docs/storing-querying/querying-data/index.md

```diff
@@ -0,0 +1,294 @@
+---
+title: "Querying Snowplow data"
+sidebar_label: "Querying data"
+sidebar_position: 2
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+## Basic queries
+
+You will typically find most of your Snowplow data in the `atomic.events` table in the `snowplow` schema, unless you configured your loader to use a different location. (In case of Redshift and Postgres, there will be extra tables for [self-describing events](/docs/understanding-your-pipeline/events/index.md#self-describing-events) and [entities](/docs/understanding-your-pipeline/entities/index.md) — see [below](#self-describing-events)).
```

Comment:
In many situations we use schema name = atomic and database name = snowplow.  But not in all situations.  For example, when loading to BigQuery for BDP enterprise the table is called like `rt_pipeline_prod1.events`.  For BDP cloud we let the user pick their own schema name.

I don't know if it's helpful to put schema name and database name in this sentence.  It might be better to say "find most of your Snowplow data in the `events` table".